packages/preview/gruvbox/1.0.0/typst.toml

```diff
@@ -0,0 +1,12 @@
+#schema: https://www.schemastore.org/typst.json
+[package]
+name = "gruvbox"
```

Comment:
Could you change this name to something less canonical? We try to avoid too descriptive names, to allow multiple packages to coexist without one having an unfair advantage. See our naming rules: https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules